readme.md

```diff
@@ -190,6 +189,8 @@ And [many more…](source/content.css)
 - ~~[Permalinked comments are highlighted.](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png)~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-26-highlighting-of-permalinked-comments/)*
 - ~~Leave a single comment in pull request diffs instead of starting a review by pressing <kbd>shift</kbd> <kbd>enter</kbd>.~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-keyboard-shortcut-to-leave-a-single-comment-in-pull-requests-reviews/)*
 
+- ~~[Quickly edit a repository's README from the repository root.](https://user-images.githubusercontent.com/170270/27501200-31a1fa20-586c-11e7-9a3f-ce270014bf0a.png)~~ *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-button-to-edit-a-repositorys-readme-from-the-repository-root/)*
```

Comment:
This should not be here.

Reply:
Hmm, wasn't this added in https://github.com/sindresorhus/refined-github/commit/a953ed4948e4dd5a73fc94413a1368b4b1c3665a?

Reply:
Yes, but it should not be part of *your* diff here. Notice how it's green.

Reply:
Hmm, I'm not sure how I can remove the diff. Any pointers?

Reply:
You just edit it out. You have it now because you resolved the merge conflict incorrectly.

Reply:
Oh, I see. Sorry about that.